code/modules/mob/living/carbon/alien/humanoid/humanoid.dm

```diff
@@ -166,4 +166,10 @@ proc/alien_type_present(var/alienpath)
 		if(!A.key || A.stat == DEAD) //Only living aliens with a ckey are valid.
 			continue
 		return 1
-	return 0
\ No newline at end of file
+	return 0
+
+
+/mob/living/carbon/alien/humanoid/check_breath(datum/gas_mixture/breath)
+	if(breath && breath.total_moles() > 0 && !sneaking)
+		playsound(get_turf(src), pick('sound/voice/lowHiss2.ogg', 'sound/voice/lowHiss3.ogg', 'sound/voice/lowHiss4.ogg'), 50, 0, -5)

```

Comment:
Shouldn't the volume be lower? Also, make it enforce a channel so that all hisses overwrite each other instead of overlapping. Or do you want a room full of xenos to sound like hissing ear rape?